CONTRIBUTING.md

```diff
@@ -0,0 +1,206 @@
+# Contributing
+
+**🎉 First off, thanks for contributing! 🎉**
+
+- [✨ Types of contributions](#types-of-contributions)
+- [📤 Pull Requests](#pull-requests)
+- [🚀 Development Workflow](#development-workflow)
+- [🔀 Release process](#release-process)
+- [🔬 Testing](#testing)
+- [🎨 Code style](#code-style)
+- [📄 Contributor license agreement](#contributor-license-agreement)
+
+## ✨ Types of contributions
+
+We welcome all types of contributions, including bug fixes, feature enhancements,
+bug reports, documentation, graphics, and many others.  You might consider contributing by:
+
+- Report a bug or request a new feature in our [issue tracker](https://github.com/DataONEorg/REPONAME/issues)
```

Comment:
I assume we want to link to https://github.com/NCEAS/vegbank2/issues

Reply:
yes, will push shortly